source/geh_common/src/geh_common/testing/spark/spark_test_session.py

```diff
@@ -70,7 +70,7 @@ def get_spark_test_session(
     else:
         master = "local[*]"
 
-    spark = builder.master(master).getOrCreate()
+    spark = builder.enableHiveSupport().master(master).getOrCreate()
```

Comment:
I'm not sure about the cons regarding using hive. Should we add an optional arg `use_hive=False`?